{"url": "https://api.github.com/repos/pytorch/pytorch/issues/comments/323045917", "html_url": "https://github.com/pytorch/pytorch/pull/2471#issuecomment-323045917", "issue_url": "https://api.github.com/repos/pytorch/pytorch/issues/2471", "id": 323045917, "node_id": "MDEyOklzc3VlQ29tbWVudDMyMzA0NTkxNw==", "user": {"login": "soumith", "id": 1310570, "node_id": "MDQ6VXNlcjEzMTA1NzA=", "avatar_url": "https://avatars0.githubusercontent.com/u/1310570?v=4", "gravatar_id": "", "url": "https://api.github.com/users/soumith", "html_url": "https://github.com/soumith", "followers_url": "https://api.github.com/users/soumith/followers", "following_url": "https://api.github.com/users/soumith/following{/other_user}", "gists_url": "https://api.github.com/users/soumith/gists{/gist_id}", "starred_url": "https://api.github.com/users/soumith/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/soumith/subscriptions", "organizations_url": "https://api.github.com/users/soumith/orgs", "repos_url": "https://api.github.com/users/soumith/repos", "events_url": "https://api.github.com/users/soumith/events{/privacy}", "received_events_url": "https://api.github.com/users/soumith/received_events", "type": "User", "site_admin": false}, "created_at": "2017-08-17T11:38:07Z", "updated_at": "2017-08-17T11:38:07Z", "author_association": "MEMBER", "body_html": "<p>thanks for figuring this out. i myself was wondering how to fix the issue you've reported</p>", "body_text": "thanks for figuring this out. i myself was wondering how to fix the issue you've reported", "body": "thanks for figuring this out. i myself was wondering how to fix the issue you've reported "}